package NDnnProto;


message TUnitNormConst {
    optional double Unit = 1;
    optional double Power = 2;
}

message TUnitNormState {
}

